packages/charts/examples/components/demo/advanced-usage/events/chart-docs-events.component.html

```diff
@@ -125,6 +125,14 @@ <h4><a href="../miscellaneous/variables.html#CHART_VIEW_STATUS_EVENT" target="_b
     <code><a href="../interfaces/IChartViewStatusEventPayload.html" target="_blank">IChartViewStatusEventPayload</a></code>.
 </p>
 
+<h4><a href="../miscellaneous/variables.html#XY_GRID_AXES_OPACITY_EVENT" target="_blank">XY_GRID_AXES_OPACITY_EVENT</a></h4>
```

Comment:
It would be nice to add this to the event sampler. But, adding a legend will conflict with @madelyneh's work for the "series visibility" event docs.